### ISSUE TITLE
Attachment Functionality needs some love

### ISSUE THREAD

**Issue by ShawnTuatara** [ahmadnassri/app-restconsole#85]
The attachments section needs to allow for individual file specification. Right now there is no way to upload multiple files each with their own custom name. My attempt at that given the features on the tool are to select 3 files in the file open dialog. Then on the "parameters key" section to specify "file1, file2, file3" but then looking at the request payload I see that each file got the name of the whole string of "file1, file2, file3". Having the ability to "add" a file to the request and that having a single file selection along with a name would make this tool functional for managing web services that allow for multipart form uploads. Having to manually change each files name to be the name of the form-data is not convenient as well as I would lose the extension and as such the auto-detection of the content-type for that form-data.

**Comment by ahmadnassri** [comment]
@ShawnTuatara you are correct, this was ignored when I first built the feature, as it wasn't something a lot of people used... but it is needed, and will be added to the next release.
